.rubocop.yml

```diff
@@ -130,6 +131,7 @@ Naming/FileName:
 Performance/HashEachMethods:
   Exclude:
     - lib/audit/catalog_to_moab.rb # it's a tiny config object; code is clearer as is
+    - lib/audit/checksum.rb #it's a tiny config object; code is clearer as is
```

Comment:
missing a space after # but I don't really care.  I'm just a little OCD.

Reply:
ha, this also triggered my OCD, despite the fact that lack of space used to be my default comment style.  rubocop has assimilated me.